tests/csrf_tests/tests.py

```diff
@@ -693,6 +693,19 @@ def test_process_response_get_token_used(self):
         ensure_csrf_cookie_view(req)
         self.assertTrue(req.session.get(CSRF_SESSION_KEY, False))
 
+    def test_session_modify(self):
+        """The session should not be modified if not necessary."""
```

Comment:
Avoid the conditionals woridng-- _The session is not modified if not necessary._